Tasks/NuGetInstaller/task.loc.json

```diff
@@ -90,7 +90,7 @@
       "groupName": "advanced",
       "options": {
         "3.3.0": "3.3.0",
-        "3.5.0.1829": "3.5.0 – build 1829 (rc2)",
+        "3.5.0.1829": "3.5.0 - build 1938 (rc2)",
```

Comment:
yep.  notice we changed the variable name from 'version' to 'uxOption' to help avoid confusion here.